INSTALL.md

```diff
@@ -1,108 +1,85 @@
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
-
-PyBitmessage can be run in either one of two ways:
-
-- straight from source
-
- or
-- from an installed
-package.
-
-## Dependencies
+### If checkdeps fails, Install dependencies manually
 Before running PyBitmessage, make sure you have all the necessary dependencies
 installed on your system.
 
-Here's a list of dependencies needed for PyBitmessage
-- python2.7
-- python2-qt4 (python-qt4 on Debian/Ubuntu)
-- openssl
-- (Fedora & Redhat only) openssl-compat-bitcoin-libs
+These dependencies may not be available on a recent OS and PyBitmessage may not build on such systems.
+Here's a list of dependencies needed for PyBitmessage based on operating system
 
-## Running PyBitmessage
-PyBitmessage can be run in two ways:
-- straight from source
-
- or
-- via a package which is installed on your system. Since PyBitmessage is Beta, it is best to run
-PyBitmessage from source, so that you may update as needed.
-
-#### Updating
-To update PyBitmessage from source (Linux/OS X), you can do these easy steps:
+For Debian-based (Ubuntu, Raspbian, PiBang, others)
+```
+python2.7 openssl libssl-dev git python-msgpack python-qt4 python-six
+```
+For Arch Linux
+```
+python2 openssl git python2-pyqt4 python-six
 ```
-cd PyBitmessage/src/
-git fetch --all
-git reset --hard origin/master
-python bitmessagemain.py
+For Fedora
+```
+python python-qt4 git openssl-compat-bitcoin-libs python-six
+```
+For Red Hat Enterprise Linux (RHEL)
+```
+python python-qt4 git openssl-compat-bitcoin-libs python-six
+```
+For GNU Guix
+```
+python2-msgpack python2-pyqt@4.11.4 python2-sip openssl python-six
 ```
-Voilà! Bitmessage is updated!
 
-#### Linux
-To run PyBitmessage from the command-line, you must download the source, then
+To run PyBitmessage from the command-line (this isn't recommended)
 run `src/bitmessagemain.py`.
 ```
-git clone git://github.com/Bitmessage/PyBitmessage.git
 cd PyBitmessage/ && python src/bitmessagemain.py
 ```
 
-That's it! *Honestly*!
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. 
 
-#### Windows
-On Windows you can download an executable for Bitmessage
-[here](https://github.com/Bitmessage/PyBitmessage/releases/download/0.6.3.2/Bitmessage_x86_0.6.3.2.exe).
+There are 3 options for running setuptools: root, user, venv
 
-However, if you would like to run PyBitmessage via Python in Windows, you can
-go [here](https://bitmessage.org/wiki/Compiling_instructions#Windows) for
-information on how to do so.
-
-#### OS X
-First off, install Homebrew.
+### as root:
 ```
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+python setup.py install
+/usr/local/bin/pybitmessage
```

Comment:
Prefix is not always `/usr/local`. For example on my PC it would be `/usr/bin/pybitmessage`. In any case it usually in `PATH`, so you can do just
```shell
$ pybitmessage
```

Reply:
i think it's better to say `PATH/pybitmessage`

Reply:
No, it has no sense:
```sh
$ echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/11/bin
$ echo PATH/pybitmessage
PATH/pybitmessage
```

Reply:
```sh
$ LC_ALL="en_US.utf8"; $PATH/pybitmessage
bash: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/11/bin/pybitmessage: No such file or directory
```